install_files/securedrop-config/DEBIAN/postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# postinst script for securedrop-config
+
+set -e
+set -x
+
+case "$1" in
+    configure)
+
+    # Ensure official Tor repo entry is removed, so that only FPF mirror is used.
+    rm -f /etc/apt/sources.list.d/deb_torproject_org_torproject_org.list
+
+    # Repoint tor repositories to FPF mirror
+    apt_security_list="/etc/apt/security.list"
+    if [ -f "$apt_security_list" ]; then
+        sed -i 's/deb\.torproject\.org\/torproject\.org/tor-apt.freedom.press/g' "$apt_security_list"
```

Comment:
won't this add a duplicate apt source once the `securedrop-config` installs ?
* a copy in `/etc/apt/security.list` and
* a revision in `/etc/apt/sources.list.d/tor.apt.freedom.press.list` 

I thought apt would throw a warning for this type of thing ... I have to confirm this scenario actually exists first though

Reply:
`/etc/apt/security.list` is the list of repos referenced by `cron-apt` for nightly upgrades. The `/etc/apt/sources.list.d/tor.apt.freedom.press.list` location is where system apt finds the repository information for normal apt install tasks. You'll also see the FPF repo (`apt.freedom.press`) duplicated in `security.list`.